__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -6,7 +6,7 @@ const details = {
   taskID: '6KhcLU3yr45dzjQIVm0J',
 };
 
-describe('TaskDetails Page', () => {
+describe.skip('TaskDetails Page', () => {
```

Comment:
why are we skipping this test ?

Reply:
This is a temporary fix, to skip these failing tests, which I will address in a dedicated PR in the future 👍🏻

Reply:
Request you to mention the issue ticket.

Reply:
https://github.com/Real-Dev-Squad/website-status/issues/504